tests/konflux-demo/config/types.go

```diff
@@ -14,8 +14,8 @@ type TestSpec struct {
 	// Name of the application created in the cluster
 	ApplicationName string `yaml:"applicationName"`
 
-	// Set of components with own specs
-	Components []ComponentSpec `yaml:"components"`
+	// Component spec
```

Comment:
```suggestion
	// Specification of the Component associated with the Application
```
or
```suggestion
	// Specification of the Component
```
too much? :sweat_smile: